src/commands/fire.yml

```diff
@@ -73,5 +79,5 @@ steps:
   - run:
       name: start firecracker
       command: |
-        docker run --volumes-from firecracker adoptopenjdk:11-jre-hotspot java --illegal-access=deny -jar /app/practitest-firecracker-standalone.jar --reports-path="app/<<parameters.test_folder>>" --api-uri=<<parameters.api_uri>> --author-id=<<parameters.author_id>> --config-path="app/config.json" <<parameters.extra_params>> create-and-populate-testset
+        docker run --volumes-from firecracker adoptopenjdk:11-jre-hotspot java --illegal-access=deny -jar /app/practitest-firecracker-standalone.jar --reports-path="app/<<parameters.test_folder>>" --api-uri=<<parameters.api_uri>> --author-id="<<parameters.author_id>>" --config-path="app/config.json" <<parameters.extra_params>> create-and-populate-testset
```

Comment:
I think something is messed up on this line (some values are quoted, some aren't)
Shouldn't we just enclose all param values in double-quotes for consistency? Unless it's a number

Reply:
some of them are number

Reply:
I was trying to change them to string so it will support to be set as Circlci Parameters but something not working with it in the code